### PR TITLE
docs: remove install command

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
Remove install command 

related: 
- fixes https://github.com/eslint/eslint/issues/16050

#### Is there anything you'd like reviewers to focus on?

lets take it from this :) feel free to suggest changes

### CODE REVIEW

**Review by aladdin-add — CHANGES_REQUESTED** [review]
can you also update it in the readme?

**Review by snitin315 — COMMENTED** [review]
Can you fix conflicts? Otherwise, LGTM

**Comment by snitin315** [comment]
Also you can use keywords like `Fix`, `Resolve`, `Closes`,  etc when linking the issue it will close the issue automatically when PR is merged.

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks!

**Review by snitin315 — APPROVED** [review]
LGTM, not sure about merging since the release-monitor check is pending.

File: README.md [hunk 1]
```diff
@@ -45,16 +45,14 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
...
-You can install ESLint using npm:
+The easiest way to initialize ESLint is:
 
 ```shell
-npm install eslint --save-dev
-```
+npm init @eslint/config
 
-You should then set up a configuration file:
+# or
```
Comment (nzakas) [inline README.md:53]: I think we can leave off Yarn now

File: README.md [hunk 2]
```diff
@@ -45,16 +45,14 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 
 Prerequisites: [Node.js](https://nodejs.org/) (`^12.22.0`, `^14.17.0`, or `>=16.0.0`) built with SSL support. (If you are using an official Node.js distribution, SSL is always built in.)
 
-You can install ESLint using npm:
+The easiest way to initialize ESLint is:
```
Comment (nzakas) [inline README.md:48]:
```suggestion
You can install and configure ESLint using this command:
```

File: docs/src/user-guide/getting-started.md [hunk 3]
```diff
@@ -20,17 +20,7 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
...
-```shell
-npm install eslint --save-dev
-
-# or
-
-yarn add eslint --dev
-```
-
-You should then set up a configuration file, and the easiest way to do that is:
+The easiest way to initialize ESLint is:
```
Comment (nzakas) [inline docs/src/user-guide/getting-started.md:23]:
```suggestion
You can install and configure ESLint using this command:
```

**Review by nzakas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (harish-sethuraman) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (harish-sethuraman) [reply README.md]: Done

File: docs/src/user-guide/getting-started.md (hunk 3 above)
Comment (harish-sethuraman) [reply docs/src/user-guide/getting-started.md]: Done

**Review by nzakas — APPROVED** [review]
LGTM. In general we don’t merge things when the release monitor is in the pending state unless it is specifically for a patch release.

**Pull request merged**